features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/LifeCycle/CheckboxLabels.jsx

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { makeStyles } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import Paper from '@material-ui/core/Paper';
+import CloseIcon from '@material-ui/icons/Close';
+import CheckIcon from '@material-ui/icons/Check';
+import green from '@material-ui/core/colors/green';
+import LaunchIcon from '@material-ui/icons/Launch';
+
+const useStyles = makeStyles(theme => ({
+    buttonsWrapper: {
+        marginTop: 40,
+    },
+    stateButton: {
+        marginRight: theme.spacing.unit,
+    },
+    paperCenter: {
+        padding: theme.spacing.unit * 2,
+        display: 'block',
+        alignItems: 'left',
+        justifyContent: 'center',
+    },
+    subHeading: {
+        fontSize: '1rem',
+        fontWeight: 400,
+        display: 'inline-block',
+    },
+    heading: {
+        fontSize: '1rem',
```

Comment:
😑 Shall we use the [`Typography`](https://material-ui.com/customization/typography/#typography) instead, Otherwise headers in different pages look different

Reply:
fixed